Library/Formula/mplayer.rb

```diff
@@ -50,19 +39,14 @@ def install
     # It turns out that ENV.O1 fixes link errors with llvm.
     ENV.O1 if ENV.compiler == :llvm
 
-    # we disable cdparanoia because homebrew's version is hacked to work on OS X
-    # and mplayer doesn't expect the hacks we apply. So it chokes.
```

Comment:
This is the relevant commit 25051afceb004091715678468b28d425d01f7591. It doesn't look like it's an option thing so I'm not really sure. I think it's probably worth leaving in unless we know it's working. CC @mistydemeo for thoughts as she added the original commit.